cmd/crossplane/core/core.go

```diff
@@ -237,6 +239,9 @@ func (c *startCommand) Run(s *runtime.Scheme, log logging.Logger) error { //noli
 	if !c.EnableCompositionFunctionsExtraResources {
 		log.Info("Extra Resources are GA and cannot be disabled. The --enable-composition-functions-extra-resources flag will be removed in a future release.")
 	}
+	if !c.WebhookEnabled {
+		log.Info("The --webhooks-enabled flag is deprecated, it is replaced with --enable-webhooks and will be removed in a future release.")
```

Comment:
i think the old flag name is `--webhook-enabled` (not plural), so this message should use that

Reply:
Fixed